### PR TITLE
(googlechrome-editthiscookie) Added new extension

### PR DESCRIPTION
**Pull request by gep13**

### CODE REVIEW

**Comment by majkinetor** [comment]
Is there any reason why chrome extensions do not have its own installer a la fosshub ?

**Comment by gep13** [comment]
No reason that I can think of. I just hadn't thought of it. But now you mention it, that would make a lot of sense.

**Comment by majkinetor** [comment]
The same goes for VStudio exts, and probalby for any other system that has plugins / exts.

Perhaps we should create a new extension that will have bunch of specialized installers, similar to core.

For example:

`Chocolatey-SpecialInstallers.psm1`

File: manual/googlechrome-extensions/googlechrome-editthiscookie/googlechrome-editthiscookie.nuspec [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>googlechrome-editthiscookie</id>
+    <version>1.0.0</version>
+    <title>EditThisCookie</title>
+    <authors>Francesco Capano</authors>
+    <owners>gep13 chocolatey</owners>
```
Comment (pascalberger) [inline manual/googlechrome-extensions/googlechrome-editthiscookie/googlechrome-editthiscookie.nuspec]: Owners should be a comma separated list as per [.nuspec Reference](https://docs.nuget.org/ndocs/schema/nuspec)

File: manual/googlechrome-extensions/googlechrome-editthiscookie/googlechrome-editthiscookie.nuspec [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    <projectUrl>http://www.editthiscookie.com/</projectUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/315c05f1387afb2acc1f028fd3752af86a9663f8/icons/googlechrome-editthiscookie.png</iconUrl>
+    <projectSourceUrl>https://github.com/fcapano/Edit-This-Cookie</projectSourceUrl>
+    <docsUrl>http://www.editthiscookie.com/start/</docsUrl>
+    <bugTrackerUrl>https://github.com/fcapano/Edit-This-Cookie/issues</bugTrackerUrl>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/manual/googlechrome-extensions/googlechrome-editthiscookie</packageSourceUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <licenseUrl>https://github.com/fcapano/Edit-This-Cookie/blob/master/License.txt</licenseUrl>
+    <description>
```
Comment (pascalberger) [inline manual/googlechrome-extensions/googlechrome-editthiscookie/googlechrome-editthiscookie.nuspec]: Stick to [Informative description guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#123-informative-description) (Use `##Features` and `##Notes`)

File: manual/googlechrome-extensions/googlechrome-editthiscookie/googlechrome-editthiscookie.nuspec (hunk 1 above)
Comment (majkinetor) [inline manual/googlechrome-extensions/googlechrome-editthiscookie/googlechrome-editthiscookie.nuspec]: You are pedantic. I like that :)

**Comment by gep13** [comment]
>@majkinetor said...
Perhaps we should create a new extension that will have bunch of specialized installers, similar to core.
For example:
Chocolatey-SpecialInstallers.psm1

Yip, this makes sense to me! 👍

**Comment by gep13** [comment]
https://github.com/chocolatey/chocolatey-coreteampackages/issues/500

**Comment by gep13** [comment]
@pascalberger thanks for the review.  I have implemented the feedback.

File: manual/googlechrome-extensions/googlechrome-editthiscookie/googlechrome-editthiscookie.nuspec [hunk 3]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>googlechrome-editthiscookie</id>
+    <version>1.0.0</version>
+    <title>EditThisCookie</title>
+    <authors>Francesco Capano</authors>
+    <owners>gep13, chocolatey</owners>
```
Comment (pascalberger) [inline manual/googlechrome-extensions/googlechrome-editthiscookie/googlechrome-editthiscookie.nuspec]: Small one: chocolatey should be listed first according to [§1.2.4](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#124-add-chocolatey-among-owners)

Comment (gep13) [inline manual/googlechrome-extensions/googlechrome-editthiscookie/googlechrome-editthiscookie.nuspec]: Great catch, done!

**Pull request merged**